packages/smooth_app/lib/database/local_database.dart

```diff
@@ -1,19 +1,15 @@
 import 'dart:async';
 
 import 'package:flutter/material.dart';
-import 'package:path/path.dart';
+import 'package:hive/hive.dart';
```

Comment:
The analyzer is complaining about this line  because `all of the used elements are also provided by hive_flutter`